commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {extends: ['@commitlint/config-conventional']};
```

Comment:
Isn't this the default?

Reply:
@anehx According to the [docs](https://commitlint.js.org/#/guides-local-setup?id=install-commitlint) this is needed.